skiko/build.gradle.kts

```diff
@@ -12,6 +14,47 @@ plugins {
     id("org.gradle.crypto.checksum") version "1.4.0"
 }
 
+fun createDownloadTasks() {
```

Comment:
Is it possible to keep this function in buildSrc?  The motivation is to limit the growth of build.gradle.kts

Reply:
Totally makes sense, renamed it to `declareSkiaTasks` and moved to the SkikoProjectContext (which belongs to buildSrc)